lib/puppet/type/cisco_bgp_neighbor_af.rb

```diff
@@ -387,6 +389,24 @@ def self.title_patterns
     newvalues(:true, :false, :default)
   end
 
+  newproperty(:route_map_in) do
+    desc 'route-map in state. Valid values are a string defining the name ' \
```

Comment:
I think you miss my point. Typecasting to string is a good thing, but an integer can be a valid route-map name so the cli won't reject it. I don't think the string check is necessary - can you give me an example where to_s does not solve this problem?